### PR TITLE
Disable spellcheck on package <input>

### LINKED ISSUE #163: Disable spellcheck on package <input>
A tiny issue that has affected me a few times when using [npmcharts.com](https://npmcharts.com/).

Some browsers (e.g., ones based on Chromium for macOS) enable [spellcheck](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/spellcheck) by default for text `<input>` elements, causing unwanted behavior like autocorrection and red dotted underline.

before-optimized

Disabling spellcheck on [package `<input>`](https://github.com/cheapsteak/npmcharts.com/blob/bfe688f6516fe00c35168291727f0a807b27ca72/packages/frontend/src/packages/packages.html#L2) improves the user experience.

### PR DESCRIPTION
**Pull request by holbrookdev**
Fixes #163

### CODE REVIEW

File: packages/frontend/src/home/contributors.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "location": "🌍"
   },
   {
-    "name": "Xiaohanxu1996",
+    "name": "xiaohanxu-nick",
```
Comment (cheapsteak) [inline packages/frontend/src/home/contributors.json:47]: this change seems unrelated?

**Review by cheapsteak — COMMENTED** [review]
Thank you🙏 

one diff to confirm

File: packages/frontend/src/home/contributors.json (hunk 1 above)
Comment (holbrookdev) [reply packages/frontend/src/home/contributors.json:47]: @cheapsteak My pleasure. Yes, that diff is unrelated. While adding myself to the contributors list, I noticed `Xiaohanxu1996` changed username on GitHub to `xiaohanxu-nick`. Seemed appropriate to update that but maybe it should have been a separate PR?

Comment (holbrookdev) [reply packages/frontend/src/home/contributors.json:47]: Confirm here: [https://github.com/cheapsteak/npmcharts.com/graphs/contributors](https://github.com/cheapsteak/npmcharts.com/graphs/contributors)

Comment (cheapsteak) [reply packages/frontend/src/home/contributors.json:47]: ah, thank you 🙏
